.github/workflows/scripts/slack_notifications/instrumented_gems.txt

```diff
@@ -0,0 +1,59 @@
+activerecord
+activesupport-broadcast_logger
+activesupport-logger
+activemerchant
+async-http
+aws-sdk-firehose
+aws-sdk-kinesis
+aws-sdk-lambda
+aws-sdk-sqs
+bunny
+capistrano
+concurrent-ruby
+curb
+dalli
+delayed_job
+dynamodb
+elasticsearch
+ethon
+excon
+fiber
+grape
+grpc
+grpc_server
+http
+httpclient
+httprb
+httpx
+kafka
```

Comment:
I don't think we have any instrumentation for a gem named kafka; just ruby-kafka and rdkafka
```suggestion
```